components/tyk-gateway/values.yaml

```diff
@@ -249,6 +249,9 @@ gateway:
   ##     readOnly: true
   extraVolumeMounts: []
 
+  # analyticsEnabled property is used to enable or disable sending analytics to Control Plane.
```

Comment:
`TYK_GW_ENABLEANALYTICS` controls if the analytics will be recorded or not. Reference to docs https://tyk.io/docs/tyk-oss-gateway/configuration/#enable_analytics

Can you update the comment accordingly?